src/m365/cli/commands/cli-doctor.spec.ts

```diff
@@ -83,7 +83,7 @@ describe(commands.DOCTOR, () => {
     sinon.stub(os, 'release').returns('10.0.19043');
     sinon.stub(packageJSON, 'version').value('3.11.0');
     sinon.stub(process, 'version').value('v14.17.0');
-    sinon.stub(auth.connection, 'appId').value('31359c7f-bd7e-475c-86db-fdb8c937548e');
+    auth.connection.appId = '31359c7f-bd7e-475c-86db-fdb8c937548e';
```

Comment:
Why did you write it like this? You're now overwriting the actual value in the appId property on runtime.

Reply:
On runtime of a test, right? This change is needed to accommodate the fact that appId is not always known, because we no longer hardcode it.

Reply:
You're correct indeed.